documentation/modules/configuring/con-config-security-providers.adoc

```diff
@@ -0,0 +1,149 @@
+The Baseline Provider is enabled by default if you don't specify a provider.
+Though you can enable it explicitly by setting the `STRIMZI_POD_SECURITY_PROVIDER_CLASS` environment variable to `baseline` when xref:ref-operator-cluster-str[configuring the Cluster Operator]. 
+
+.Configuration for the Baseline Provider
+[source,yaml,options="nowrap"]
+----
+# ...
+env:
+  # ...
+  - name: STRIMZI_POD_SECURITY_PROVIDER_CLASS
+    value: baseline
+  # ...
+----
+
+Instead of specifying `baseline` as the value, you can specify the `io.strimzi.plugin.security.profiles.impl.BaselinePodSecurityProvider` classpath.
+
+== Restricted Provider for pod security
+
+The Restricted Provider provides a higher level of security than the Baseline Provider.
+It configures the pods managed by Strimzi with a restricted security profile.
+
+You enable the Restricted Provider by setting the `STRIMZI_POD_SECURITY_PROVIDER_CLASS` environment variable to `restricted` when xref:ref-operator-cluster-str[configuring the Cluster Operator].
+
+.Configuration for the Restricted Provider
+[source,yaml,options="nowrap"]
+----
+# ...
+env:
+  # ...
+  - name: STRIMZI_POD_SECURITY_PROVIDER_CLASS
+    value: restricted
+  # ...
+----
+
+Instead of specifying `restricted` as the value, you can specify the `io.strimzi.plugin.security.profiles.impl.RestrictedPodSecurityProvider` classpath.
+
+If you change to the Restricted Provider from the default Baseline Provider, the following restrictions are implemented in addition to the constraints defined in the baseline security profile:
+
+* Limits allowed volume types
+* Disallows privilege escalation
+* Requires applications to run under a non-root user 
+* Requires `seccomp` (secure computing mode) profiles to be set as `RuntimeDefault` or `Localhost`
+* Limits container capabilities to use only the `NET_BIND_SERVICE` capability
+
+With the Restricted Provider enabled, containers created by the Cluster Operator are set with the following security context.
+
+.Cluster Operator with restricted security context configuration
+[source,yaml,options="nowrap"]
+----
+# ...
+securityContext:
+  allowPrivilegeEscalation: false
+  capabilities:
+    drop:
+      - ALL
+  runAsNonRoot: true
+  seccompProfile:
+    type: RuntimeDefault
+# ...
+----
+
+[NOTE]
+====
+Container capabilities and `seecomp` are Linux kernel features that support container security. 
+
+* Capabilities add fine-grained privileges for processes running on a container. The `NET_BIND_SERVICE` capability allows non-root user applications to bind to ports below 1024. 
+* `seccomp` profiles limit the processes running in a container to only a subset of system calls.  
+The `RuntimeDefault` profile provides a default set of system calls.
+A `LocalHost` profile uses a profile defined in a file on the node.   
+
+====
+
+[role="_additional-resources"]
+.Additional resources
+
+* {K8sSecurityContext} on Kubernetes
+* {K8sSecurityStandards} on Kubernetes (including profile descriptions)
```

Comment:
The relation to the security standars is quite important. Because users will get the requirement to run under the restricted standard. So I wonder if we can make the docs a bit more clear about the relationship between these. Right now it looks like it is just this link.

Reply:
I've added a couple of descriptions where the association between provider and profile is explicit. And another link to the Kubernetes doc.